sinks/api/metricdecoder.go

```diff
@@ -121,24 +123,36 @@ func (self *defaultDecoder) getContainerMetrics(container *source_api.Container,
 				} else {
 					startTime = stat.Timestamp
 				}
-				points := supported.GetValue(&container.Spec, stat)
-				for _, point := range points {
+				seriesName := supported.Name
+				if supported.Units.String() != "" {
+					seriesName = fmt.Sprintf("%s_%s", seriesName, supported.Units.String())
```

Comment:
I don't think GCM will like diverging from the registered series names. What ever series name we used while registering the events is what we should use here.
If Influxdb decides to mangle the series name with additional metadata, then it must be local to Influxdb.